goosepaper/multiparser.py

```diff
@@ -27,35 +37,150 @@ def __init__(self):
+        )
         self.parser.add_argument(
             "--replace",
             action="store_true",
             required=False,
             default=None,
-            help="Will replace a document with same name in your remarkable.",
+            help="||Will replace a document with same name in your remarkable.\nDefault behaviour is case sensitive (ala *nix/Mac).\n\ne.g. 'A Flock of RSS Feeds.epub' and 'a flock of rss feeds.epub'\nare seen as TWO different files. Can be altered with '--nocase'\nor '--strictlysane' switches.",
         )
         self.parser.add_argument(
-            "-f",
-            "--folder",
+            "--noreplace",
+            action="store_true",
             required=False,
-            help="Folder to which the document will be uploaded in your remarkable.",
+            default=None,
+            help="Only valid when specified on command line (ignored if present in any config file). Supersedes any config file setting for 'replace: true', thus ensuring that the file will NEVER be overwritten. Will also supersede command line '--replace' if both are specified regardless of order."
+            )
+        self.parser.add_argument(
+            "--cleanup",
+            required=False,
+            default=None,
+            action="store_true",
+            help="Delete the output file after upload."
+        )
+        self.parser.add_argument(
+            "--nocase",
+            required=False,
+            default=None,
+            action="store_true",
+            help="** COMING SOON: Enables case insensitivity on upload. By default folders \"My Goosepapers\" and \"my goosepapers\" are NOT the same folder. Enabling this feature means that they would be considered the same. No effect unless \"-u\" is specified."
+        )
+        self.parser.add_argument(
+            "--strictlysane",
+            required=False,
+            default=None,
+            action="store_true",
+            help="** COMING SOON: Force strict sanity checking, implies \"--no-case\". Fail on ANY duplicate folder or document names in the root or target upload folders. No effect unless \"-u\" is specified."
         )
         self.args = self.parser.parse_args()
 
-        try:
-            self.config = load_config_file(self.args.config)
-        except FileNotFoundError as e:
-            raise FileNotFoundError(
-                f"Could not find the configuration file at {self.args.config}"
-            ) from e
+        # These are in order of precedence, low to highest
+        #  1. Home directory global configs
+        #  2. Local directory from which goosepaper is called
+        #  3. Specified on the command line.
+        
+        defaultconfigs = [ os.getenv('HOME') + "/.goosepaper.json", './goosepaper.json', self.args.config ]
+        foundconfig = False
+        self.config = {}
+        outputcount=0
```

Comment:
I think this isn't styled according to the `black` styleguides. I don't have that linting set up automatically anywhere, but it might be useful to run [`black`](https://github.com/psf/black) on this branch to quickly clean up operator spacing things like this :)

Reply:
Fixed.